source/tests/SGM/SGMMonochromatorInfoTestView.h

```diff
@@ -0,0 +1,75 @@
+#ifndef SGMMONOCHROMATORINFOTESTVIEW_H
+#define SGMMONOCHROMATORINFOTESTVIEW_H
+
+#include <QWidget>
+#include <QTextEdit>
+#include <QDoubleSpinBox>
+#include <QLabel>
+#include <QComboBox>
+#include <QCheckBox>
+#include "beamline/SGM/monochromator/SGMGratingSupport.h"
+#include "beamline/SGM/monochromator/SGMUndulatorSupport.h"
+class SGMMonochromatorInfo;
+class SGMMonochromatorInfoTestView : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit SGMMonochromatorInfoTestView(SGMMonochromatorInfo* mono, QWidget *parent = 0);
+
+signals:
+
+public slots:
+
+protected slots:
+    void onMonoGratingAngleChanged(double gratingAngleSteps);
```

Comment:
A couple comments for these guys would be a nice addition.  Nothing crazy since they are fairly straight-forward.